File: 1060.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
var s = 0
for(var n1 = 1; n1 < 7; n1 = n1 + 1){
    var numero = Number(lines.shift());
    if(numero > 0){
        var s = s+1}}
console.log(`${s} valores positivos`);

File: 1060.c
#include <stdio.h>
#include <stdlib.h>
int main(){
    int contador = 0;
    float numero;
    for(int i = 0; i < 6; i++){
        scanf("%f", &numero);
        if(numero > 0)
            contador++;
    }
    printf("%i valores positivos\n", contador);
    return 0;
}
